src/main/java/socially/disturbed/command/CommandDto.java

```diff
@@ -0,0 +1,60 @@
+package socially.disturbed.command;
```

Comment:
Liten fanesak -> i Kotlin er alle felt public.
For Java synes jeg spesielt i DTO/Data-klasser er gettere og settere kun støy.. De har ingen praktisk/designmessig betydning, og kunne vært fjernet.
Bare la det være slik - men da har jeg nevnt det! :-p

Reply:
Litt mixed signals, du bruker jo getters og setters selv enkelte steder? 

Men vi kan kjøre på med uten getters og setters :) 
Vi må bare gjøre det likt overalt da, ikke ha noe mix